pillar.example

```diff
@@ -19,3 +19,15 @@ samba_sections:
     guest ok: no
     writeable: no
     printable: yes
+  sharename:
+    comment: "This is a shared directory"
+    browseable: yes
+    writeable: yes
+    valid users: @sharegroup
+    force group: sharegroup
+    create mask: 0660
+    directory mask: 2770
+
+samba_users:
+   user1: user1sambapassword
```

Comment:
Should we nest passwords below users so it looks like:
```
samba_users:
  user1:
    password: blah
```
 I haven't worked with Samba in a while so I can't remember if there would be any other settings you might need for a user.